data/json/techniques.json

```diff
@@ -61,7 +65,8 @@
             "You swing through %s and everyone nearby",
             "<npcname> swings through %s and everyone nearby"
         ],
-        "aoe" : "spin"
+        "aoe" : "spin",
+        "description" : "Attack everyone nearby, crit only, min 4 melee"
```

Comment:
Have you tried using tags in those?
In your older descriptions, you used tags to color up the description. This increased readability somewhat.

Also, "nearby" is a bit vague. "Adjacent" would be better. As far as I recall, it doesn't attack allies, so "Attack adjacent enemies" would probably be better.

Reply:
> Have you tried using tags in those?
In your older descriptions, you used tags to color up the description. This increased readability somewhat.

No, I haven't, but I used tags in the .cpp. Have you seen updated pic in the OP? 
The name of the technique is blue, its description is light blue.